Tutorials/Creating-Tables-for-Umbraco-with-PetaPoco/index-v7.md

```diff
@@ -0,0 +1,248 @@
+---
+versionFrom: 7.0.0
+---
+# Tutorial - Creating Tables in Umbraco with PetaPoco
+
+## Overview
+
+This guide will take you steps to create tables in Umbraco using Peta Pocos and the ability to access that data in that table
+
+## What is Peta Poco?
+
+PetaPoco is a tiny & fast micro-ORM for . NET. It is tiny with abosoutely no dependencies
+
+## Requirements Needed to Use PetaPoco
+
+- Umbraco 7+ with a SQL Database
+- Umbraco 8+ does not support Peta Poco
+- Nuget Peta Poco Package - Optional (https://www.nuget.org/packages/PetaPoco/)
+
+## Steps to Create Your First Table
+- [Setting up the folders](#setting-up-the-folders)
+- [Creatung our first table](#creating-our-first-table)
+- [Implementing new table](#implementing-new-table)
+- [Accessing the new table](#accessing-the-new-table)
+
+## Related Links
+[CollaboratingPlatypus/PetaPoco](https://github.com/CollaboratingPlatypus/PetaPoco)
```

Comment:
What does this link to?
Would be good with some more context to this related link! :)